### PR TITLE
feat: add Centrifuge Mainnet and Centrifuge Testnet Amber

### PR DESCRIPTION
**Pull request by philipstanislaus**
Thanks for your great work on Parity Signer! We would love to use it for our mainnet and testnet ("Amber") as well - would that be possible?

### CODE REVIEW

File: src/stores/RegistriesStore.ts [hunk 1]
```diff
@@ -34,7 +34,9 @@ type NetworkTypesMap = {
 const networkTypesMap: NetworkTypesMap = {
 	centrifuge: {
 		alias: 'centrifuge-chain',
-		chains: {}
+		chains: {
+			centrifuge_amber: 'centrifuge-chain-amber'
```
Comment (hanwencheng) [inline src/stores/RegistriesStore.ts:38]: It seems this is not record in the [API](https://github.com/polkadot-js/api/tree/master/packages/types-known/src) yet?

**Comment by hanwencheng** [comment]
please also add icons to ios env, which is under `ios/assets/res` folder

File: src/stores/RegistriesStore.ts (hunk 1 above)
Comment (philipstanislaus) [reply src/stores/RegistriesStore.ts:38]: If I read the code correctly, it uses `centrifuge` as the lookup in the API's known types. `centrifuge-chain-amber` might be used at a later point as we will use it for breaking changes. Do you think we should add separate types for `centrifuge-chain-amber` already to the API?

**Comment by philipstanislaus** [comment]
Thanks for your review @hanwencheng . Just pushed the logos as well.

File: src/stores/RegistriesStore.ts (hunk 1 above)
Comment (hanwencheng) [reply src/stores/RegistriesStore.ts:38]: understood, no need to add separate types for now, the types of `centrifuge-chain` will be included with this structure. If you amber chain has breaking changes then you may add them into API to overwrite the main net's types.

**Pull request merged**